mmv1/third_party/terraform/services/accesscontextmanager/resource_access_context_manager_access_policy_test.go.erb

```diff
@@ -17,21 +17,22 @@ import (
-		"service_perimeter_update":         testAccAccessContextManagerServicePerimeter_updateTest,
-		"service_perimeter_resource":       testAccAccessContextManagerServicePerimeterResource_basicTest,
-		"access_level":                     testAccAccessContextManagerAccessLevel_basicTest,
-		"access_level_full":                testAccAccessContextManagerAccessLevel_fullTest,
-		"access_level_custom":              testAccAccessContextManagerAccessLevel_customTest,
-		"access_levels":                    testAccAccessContextManagerAccessLevels_basicTest,
-		"access_level_condition":           testAccAccessContextManagerAccessLevelCondition_basicTest,
-		"service_perimeter_egress_policy":  testAccAccessContextManagerServicePerimeterEgressPolicy_basicTest,
-		"service_perimeter_ingress_policy": testAccAccessContextManagerServicePerimeterIngressPolicy_basicTest,
-		"service_perimeters":               testAccAccessContextManagerServicePerimeters_basicTest,
-		"gcp_user_access_binding":          testAccAccessContextManagerGcpUserAccessBinding_basicTest,
-		"authorized_orgs_desc":             testAccAccessContextManagerAuthorizedOrgsDesc_basicTest,
+          "access_policy":                            testAccAccessContextManagerAccessPolicy_basicTest,
+          "access_policy_scoped":                     testAccAccessContextManagerAccessPolicy_scopedTest,
+          "service_perimeter":                        testAccAccessContextManagerServicePerimeter_basicTest,
+          "service_perimeter_update":                 testAccAccessContextManagerServicePerimeter_updateTest,
+          "service_perimeter_resource":               testAccAccessContextManagerServicePerimeterResource_basicTest,
+          "service_perimeter_dry_run_resource":       testAccAccessContextManagerServicePerimeterDryRunResource_basicTest,
+          "access_level":                             testAccAccessContextManagerAccessLevel_basicTest,
+          "access_level_full":                        testAccAccessContextManagerAccessLevel_fullTest,
+          "access_level_custom":                      testAccAccessContextManagerAccessLevel_customTest,
+          "access_levels":                            testAccAccessContextManagerAccessLevels_basicTest,
+          "access_level_condition":                   testAccAccessContextManagerAccessLevelCondition_basicTest,
+          "service_perimeter_egress_policy":          testAccAccessContextManagerServicePerimeterEgressPolicy_basicTest,
+          "service_perimeter_ingress_policy":         testAccAccessContextManagerServicePerimeterIngressPolicy_basicTest,
+          "service_perimeters":                       testAccAccessContextManagerServicePerimeters_basicTest,
+          "gcp_user_access_binding":                  testAccAccessContextManagerGcpUserAccessBinding_basicTest,
+          "authorized_orgs_desc":                     testAccAccessContextManagerAuthorizedOrgsDesc_basicTest,
```

Comment:
Can these be changed back to having tab-spacing?

Reply:
Reformatted the file. Thanks for reviewing!